docs/rules/no-child-traversal-in-connectedcallback.md

```diff
@@ -52,3 +52,5 @@ class FooBarElement extends HTMLElement {
 ## When Not To Use It
 
 If you are comfortable with the edge cases of DOM traversal directly in the `connectedCallback` then you can disable this rule.
+
+If your element is appended to the DOM via `.innerHTML` or similar methods, you may need to disable this rule because in such a case, the children are are available during `connectedCallback` and no mutations events will fire.
```

Comment:
```suggestion
If your element is appended to the DOM via `.innerHTML` or similar methods, you should disable this rule because in such a case, the children are are available during `connectedCallback` and no mutations events will fire.
```